### ISSUE TITLE
Editor Automatically Replacing Special Characters.

### ISSUE THREAD

**Issue by rwatts3** [Microsoft/vscode#53495]
- VSCode Version: Insiders 1.25
- OS Version: Win 7

Steps to Reproduce:

1. Special Characters such as ' or html characters within the editor
2. Notice changes when diffing files for svn or git even if the file was not physically altered.

Does this issue occur when all extensions are disabled?: Yes

**Comment by rwatts3** [comment]
This appears to be happening from the core editor somewhere around the sprint leading to 1.25. 
I held off on submitting the issue thinking it would be resolved. However there are several projects where a line was changed without my input.

This is certainly a problem as it's not ideal to commit line changes that i'm not intending to implement. And I am therefore having to go back and check every project i've worked on in the past weeks as well as double check just about every file prior to making a commit to ensure that lines characters were not changed without my intention.

**Comment by alexandrudima** [comment]
@rwatts3 Do you have a lot of files with mixed line endings in your projects? We normalize line endings inside a file (i.e. if some lines in the same file use CRLF and some other lines in the same file use LF we will count which one is predominant and normalize that **when you change and save the file**). But only in the case where you edit the file...

Can you include some steps to help me reproduce?

**Comment by rwatts3** [comment]
Hi I think this may be different from my initial stated problem.
But what you've just described certainly raises a concern for me. 

I work in an environment where there are numerous developers over 100 different individuals that may checkout a project and conduct their work at any given point in time. 

Not all of those developers are going to have a uniformed line ending setting. However I'm finding it really hard to believe why it would be a good idea to automatically tally up and select one without my direct approval or confirmation especially on something that will be committed and submitted to a repository and you generally do not see or notice that type of change. 

This is very dangerous and I think it should be changed. 

For example had I realized this was happening it could have solved an issue I had several weeks ago with a python script. 

My co-worker sent me a python script to run that he committed to the repository and it completely failed when I attempted to run it. He uses sublime which didn't change his line endings automatically for him. I opened his script, and for some reason the line endings were changed and it took me 2 days to discover this change. It wasn't until I opened the script in sublime that I realized it worked. 

I just don't see the benefit in changing files without a prompt for approval in the editor.
